### PR TITLE
fix (hubspot): HubSpot deprecated API Key authorization

### PR DESCRIPTION
**Pull request by leonardofurnielis**
(https://developers.hubspot.com/changelog/upcoming-api-key-sunset)

HubSpot API Key will be sunset in November 30, 2022. Update OpenAPI JSON to bearer and read.me (how to create new access token).

### CODE REVIEW

**Comment by amiddavid** [comment]
it so happen that we opened the same pull request...
if you could be so kind as to add to yours some of the data I wrote in the readme...
specifically:
1. maintain the backward docs  ( reason: till Nov30 , people can still use what they created, and if they come here... I want them to easily see things have changed and why)
2. add the link to the private apps as I did.

Otherwise your PR is awesome and thank you!

**Comment by leonardofurnielis** [comment]
@amiddavid thanks for your comments, updated!

File: integrations/extensions/starter-kits/hubspot/readme.md [hunk 1]
```diff
@@ -20,7 +20,21 @@ HubSpot has a fantastic public API documentation that is very custom extensions
...
+## Pre-Req 1: Getting Private Apps Access Token
+#### ** Updated Aug. 29, 2022 **
+
+Hubspot has increased their level of security for making API calls. They no longer support one API Key but rather support keys per Application. Hubspot now support two types of Apps: Private and Public. Public Apps require OAuth authentication which currently IBM Watson Assistant does not support. For the time being, we recommend using Private Apps which are based on Bearer Authentication. Please note, to follow Hubspot guidelines on sharing this Bearer token.
+
+To call the HubSpot API, you will need an Private App access token. To obtain this key:
+
+1. Click on `Settings` (the cog wheel icon) at the top right of the page, then click `Integrations` in the left menu to open a drop-down, and select `Private Apps`.
+
+1. If you do not already have one, `Create a private app`. Enter your app's name than, click the `Scopes` tab select the scopes `tickets` and `crm.objects.contacts.read`. Click `Create app` in the top right. Copy the access token and save for future steps.
```
Comment (amiddavid) [inline integrations/extensions/starter-kits/hubspot/readme.md:32]: I think that for instance to create a ticket you need additional scopes

Comment (leonardofurnielis) [reply integrations/extensions/starter-kits/hubspot/readme.md:32]: Hi @amiddavid I have tested with these scopes and created a ticket over WA. it worked!

**Review by amiddavid — APPROVED** [review]

**Pull request merged**